File: mpp_proiect_java/src/main/java/triatlon/domain/Rezultat.java
package triatlon.domain;

import java.time.LocalDateTime;

/***
 *
 */
public class Rezultat extends Entity<Long>{

    private Proba proba;
    private Participant participant;
    private Integer numarPuncte;


    public Rezultat(Long idRezultat,Proba proba,Participant participant, Integer numarPuncte){
        setId(idRezultat);
        this.proba = proba;
        this.participant = participant;
        this.numarPuncte = numarPuncte;

    }

    public Proba getProba() {
        return proba;
    }

    public void setProba(Proba proba) {
        this.proba = proba;
    }

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public Integer getNumarPuncte() {
        return numarPuncte;
    }

    public void setNumarPuncte(Integer numarPuncte) {
        this.numarPuncte = numarPuncte;
    }

    @Override
    public String toString() {
        return "Rezultat " + proba.getTipProba() + " " + participant.getFirstName() + " " + numarPuncte.toString();
    }
}

File: mpp_proiect_CSharp/triatlon/triatlon/domain/Rezultat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace triatlon.domain
{
    class Rezultat : Entity<long>
    {
        public long Id { get; set; }

        public Proba proba { get; set; }
        public Participant participant { get; set; }

        public int numarPuncte { get; set; }

        public Rezultat(long idrezultat1,Proba proba1, Participant participant1, int numarpuncte1)
        {
            this.Id = idrezultat1;
            this.proba = proba1;
            this.participant = participant1;
            this.numarPuncte = numarpuncte1;
            
        }

        public override string ToString()
        {
            return "Rezultat " + proba.tipProba + " " + participant.firstName + " " + numarPuncte;
        }
    }
}
